openssh/config.sls

```diff
@@ -3,17 +3,35 @@
 include:
   - openssh
 
-{% if salt['pillar.get']('sshd_config', False) %}
-sshd_config:
+{%- macro sshd_config_file(name) %}
   file.managed:
-    - name: {{ openssh.sshd_config }}
+    - name: {{ name }}
     - source: {{ openssh.sshd_config_src }}
     - template: jinja
     - user: {{ openssh.sshd_config_user }}
     - group: {{ openssh.sshd_config_group }}
     - mode: {{ openssh.sshd_config_mode }}
     - watch_in:
       - service: {{ openssh.service }}
+{%- endmacro %}
+
```

Comment:
Wouldn't it be cleaner (just for readability sake) if you move the whole macro *above* the
```yaml
include:
  - openssh
```
lines?